packages/subcommands/src/lib/Subcommand.ts

```diff
@@ -15,10 +15,40 @@ import {
 
 export class Subcommand<PreParseReturn extends Args = Args, O extends Subcommand.Options = Subcommand.Options> extends Command<PreParseReturn, O> {
 	public parsedSubcommandMappings: SubcommandMappingArray;
+	private caseInsensitiveSubCommands = false;
 
 	public constructor(context: PieceContext, options: O) {
 		super(context, options);
 		this.parsedSubcommandMappings = options.subcommands ?? [];
+
+		const clientOptions = this.container.client.options;
+		if (clientOptions.caseInsensitiveCommands) {
+			// Because slash commands must be lowercase anyways, blanketing all commands to lowercase is safe.
```

Comment:
if the user doesnt enter lowercased name it will automatically throw an error, wont that be better? instead of auto lowercasing it

Reply:
this is the behavior of the framework I think, so no